<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editemployee</class>
 <widget class="QDialog" name="editemployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>241</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="NameInforationEdit">
    <item row="3" column="0">
     <widget class="QLabel" name="Prenom">
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="NomInput"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="PrenomInput"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Nom">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="TitleName">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>&lt;b&gt; General informations &lt;/b&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>241</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="InformaticienLoginGridEdit">
    <item row="1" column="0" colspan="2">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="editInformaticienPassword">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="InformaticienPassword">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="editInformaticienLogin">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="InformaticienLogin">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="informaticienIdentification">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Computer scientist identification&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>241</width>
     <height>66</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="EmployeeTypeEdit">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;b&gt; Employee type &lt;/b&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Informaticien</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Banquier type A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Banquier type B</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Assureur logement</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Assureur voiture</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Assureur vie</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Divers</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>352</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
